components/cbaSettings/medical_basic.sqf

```diff
@@ -1,31 +1,33 @@
+// ACE Medical
 force ace_medical_ai_enabledFor = 2;
 force ace_medical_AIDamageThreshold = 1;
 force ace_medical_bleedingCoefficient = 0.4;
 force ace_medical_blood_enabledFor = 2;
+force ace_medical_enableVehicleCrashes = false;
 force ace_medical_fatalDamageSource = 0;
-force ace_medical_fractureChance = 0;
+force ace_medical_fractureChance = 0.05;
 force ace_medical_fractures = 1;
 force ace_medical_gui_enableMedicalMenu = 1;
 force ace_medical_gui_maxDistance = 3;
 force ace_medical_gui_openAfterTreatment = true;
 force ace_medical_ivFlowRate = 3.5;
 force ace_medical_limping = 1;
 force ace_medical_painCoefficient = 0.5;
-force ace_medical_playerDamageThreshold = 1.5;
+force ace_medical_playerDamageThreshold = 2.5;
 force ace_medical_spontaneousWakeUpChance = 0.9;
-force ace_medical_spontaneousWakeUpEpinephrineBoost = 1;
-force ace_medical_statemachine_AIUnconsciousness = false;
+force ace_medical_spontaneousWakeUpEpinephrineBoost = 1.5305;
+force ace_medical_statemachine_AIUnconsciousness = true;
```

Comment:
been run in all my mission lately, but if we need to push it before a committee ill gladly transfer the work to someone else since this will be my personal standard for my mishs from now on

Reply:
I'm inclined to approve these changes. The current state of this issue isn't ideal, and I think we'd benefit from taking action sooner rather than later. Worst case scenario, this still only takes a minor hotfix to revert should it suddenly turn sour despite no issues in playtesting so far.